app/components/log-in.js

```diff
@@ -0,0 +1,210 @@
+////https://github.com/EnCiv/undebate-ssp/issues/108
```

Comment:
I don't see this file being used anywhere - if not then remove it. Otherwise lets discuss it seems to be alot like the signin-signup file.